app/views/data_quality_reports/version_four/project/README.md

```diff
@@ -0,0 +1,235 @@
+
+The sum of the `UnitInventory` fields for project inventories with inventory covering any
+part of the reporting period.
+
+### Average Daily / Nightly Clients
+
+*NOTE:* Calculated from computed service records.
+
+The sum of the number of service entries for heads of households on each day of the reporting period divided by
+the number of days.
+
+### Average Unit Utilization
+
+*NOTE:* Calculated from computed service records.
+
+The average daily clients from above divided by the unit inventory from above.
+
+### Graph
+
+*NOTE:* Calculated from computed service records.
+
+The sum of the number of service entries for heads of households on each day of the reporting period.
+
+## Completeness
+
+Each attribute is assigned to the first matching bucket in the order:
+refused, not collected, missing, partial, complete.
+
+### Completeness Percentage
+
+Denominator: The number of enrollments * the number of completeness metrics)
+Numerator: Denominator - The number of metrics not marked _complete_.
+
+### Name
+
+Computed for all enrollments using `NameDataQuality`.
+Name is missing if the `FirstName` or `LastName` are blank.
+
+### SSN
+
+Computed for all enrollments using `SSNDataQuality`.
+SSN is missing if it is blank, or is not a valid SSN:
+- Cannot contain a non-numeric character
+- Must be 9 digits long
+- First three digits cannot be "000," "666," or in the 900 series
+- The second group / 5th and 6th digits cannot be "00"
+- The third group / last four digits cannot be "0000"
+- Cannot be repetitive (e.g. "333333333")
+- Cannot be one of '219099999', '078051120', or '123456789'
+
+### DOB
+
+Computed for all enrollments using `DOBDataQuality`.
+DOB is missing if it is:
+- Blank
+- Incomplete (which implies that DOB's will never be reported as partial)
+- Is before Jan 1, 1915
+- Is after the `DateCreated` of the associated enrollment
+- If the client is an adult on the report date or head of household, and the DOB is on or after the `EntryDate`
+
+### Gender
+
+Computed for all enrollments using `Gender`.
+Gender is missing if it is blank.
+
+### Veteran Status
+
+Only computed for enrollments using `VeteranStatus` if the client is an adult on the `EntryDate`.
+Veteran Status is missing if it is blank.
+
+### Ethnicity
+
+Computed for all enrollments using `Ethnicity`.
+Ethnicity is missing if it is blank.
+
+### Race
+
+Computed for all enrollments using `RaceNone`.
+Race is missing if `RaceNone`, `AmIndAKNative`, `Asian`, `BlackAfAmerican`, `NativeHIOtherPacific`, and `White`
+are all blank.
+
+### Disabling Condition
+
+Computed for enrollments using `DisablingCondition` if the client is an adult on the report date.
+Disabling Condition is missing if `DisablinghCondition` is blank, or if it is '0', and if any of the most
+recent disability responses are '1'.
+
+### Prior Living Situation
+
+Computed for enrollments where the client is either the head of household, or an adult on
+the report date, using`LivingSituation`.  
```

Comment:
this is on the entry date to the enrollment right?

Reply:
`is_adult?(date: Date.today) || head_of_household`, is this what you are asking?

Reply:
sorry, no.  What's `report date`?

Reply:
Is 'on the date the report is generated' clearer?

Reply:
hmm, I'm starting to question the `Date.today`.  I think that should be the entry date in this situation.

Reply:
I agree. To discuss on Monday?

Reply:
Monday